packages/gravity-ui-web/src/sass/06-utilities/_padding.scss

```diff
@@ -0,0 +1,11 @@
+$grav-sides: (top, bottom, left, right);
+$grav-spaces-map: ('xs': $grav-sp-s,'s': $grav-sp-s, 'm': $grav-sp-m, 'l': $grav-sp-l, 'xl': $grav-sp-xl, 'xxl': $grav-sp-xl);
```

Comment:
Should we have a zero option too for removing a padding?

Also, I wonder if it might make sense to move this map up into the settings layer. I can imagine it being useful elsewhere too - e.g. of we ever add margin util classes. You might also want to consider renaming it to `$grav-sp-map` to align woth our naming conventions.

Reply:
Yep a zero one would be useful, can add that.  Sure can move it to settings.

Reply:
Added a none value to the map